src/DynamoCore/Models/RecordableCommands.cs

```diff
@@ -970,6 +984,7 @@ where xmlNode.Name.Equals("ModelGuid")
             internal IEnumerable<Guid> ModelGuids { get { return modelGuids; } }
             internal string Name { get; private set; }
             internal string Value { get; private set; }
+            internal WorkspaceModel TargetWorkspace { get; private set; } 
```

Comment:
My first idea is to use `Guid`  instead of a `WorkspaceModel` instance, and it could be serialized. But when we serialize it, it is highly possible that we can't find the corresponding workspace model because 1) we didn't serialize workspace GUID to xml file; 2) we don't have home workspace creation command. 

What we can do here is, replace `WorkspaceModel` with `Guid`, but always serialize GUID.Empty to xml file, and update it once we support 1) and 2) mentioned before. What do you think?